formbricks/formbricks
# feat: kamal deployment integration

## What does this PR do?
- Sets up [Kamal](https://kamal-deploy.org) for deploying Formbricks on any Ubuntu/Linux Machine being cloud agnostic at the same time
- Pushed new image to the configured registry on each build & uses that to run it on our Cloud
- GitHub Action for CDs
- Traefic Configuration to work with Lets Encrypt Self Signed SSL

ToDo:
- [ ] Set GitHub Secrets for each env var

## How should this be tested?run Formbricks

Hosted URL: https://kamal.formbricks.com

1. Make a change to this branch
2. Push it
3. See the GitHub Action build & push the new image both to our registry as well as deploy it!

Refer to [.kamal/README.md ](https://github.com/ShubhamPalriwala/formbricks/blob/shubham/kamal-test/.kamal/README.md) for our Docs for setting up Kamal to run locally in your machine

## Checklist



### Required


### Appreciated
